source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3198,7 +3198,7 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
    ------------------------  -------------------------------------------------------------------------------------------------------------
    leaf node                 moves focus to tracks table
    ------------------------  -------------------------------------------------------------------------------------------------------------
-   **Tracks table**          loads selected track. See "Playing track protection" in Preferences ‣ Decks
+   **Tracks table**          Performs the "Track Double-Click Action" selected in Preferences ‣ Library. Default: load selected track. Also see "Playing track protection" in Preferences ‣ Decks
```

Comment:
Please use `guilabel` role instead of inserting triangles manually

Reply:
Or rather `:menuselection:` as documented here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html

Reply:
Yes, I used :menuselection: and also adjusted the existing menu hint.